<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perfil</string>
  </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <property name="spacing">
    <number>22</number>
  </property>
  <property name="leftMargin">
    <number>24</number>
  </property>
  <property name="topMargin">
    <number>24</number>
  </property>
  <property name="rightMargin">
    <number>24</number>
  </property>
  <property name="bottomMargin">
    <number>24</number>
  </property>
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
     <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Mi perfil</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="titleSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="titleSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
   <widget class="QFrame" name="cardFrame">
     <layout class="QVBoxLayout" name="cardLayout">
      <property name="leftMargin">
       <number>22</number>
      </property>
      <property name="topMargin">
       <number>22</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>22</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="avatarLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <item>
         <widget class="QLabel" name="avatarPlaceholder">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>140</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>160</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="chooseAvatarButton">
          <property name="text">
           <string>Cambiar foto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignRight|Qt::AlignVCenter</set>
        </property>
        <property name="horizontalSpacing">
         <number>20</number>
        </property>
        <property name="verticalSpacing">
         <number>16</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>Usuario</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="usernameValue">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="usernameHintLabel">
          <property name="text">
           <string>El usuario no se puede cambiar.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Contrase침a</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="passwordContainer">
        <layout class="QHBoxLayout" name="passwordLayout">
         <property name="spacing">
            <number>12</number>
         </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="passwordLineEdit">
             <property name="placeholderText">
              <string>Actualiza tu contrase침a</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item>
           <widget class="QToolButton" name="togglePasswordButton">
             <property name="toolTip">
              <string>Mostrar/ocultar contrase침a</string>
             </property>
             <property name="text">
              <string>Ver</string>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="emailLabel">
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="placeholderText">
           <string>Actualizar email</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="birthdateLabel">
          <property name="text">
           <string>Fecha de nacimiento</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="birthdateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="buttonsTopSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonsLayout">
        <property name="spacing">
         <number>14</number>
        </property>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="text">
           <string>Cerrar sesi칩n</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="buttonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="default">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
